test/nettests/whatsapp.cpp

```diff
@@ -0,0 +1,98 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+
+#ifdef ENABLE_INTEGRATION_TESTS
+#define CATCH_CONFIG_MAIN
+#include "private/ext/catch.hpp"
+#include "private/ooni/whatsapp.hpp"
+
+#include "../nettests/utils.hpp"
+
+using namespace mk::nettests;
+using namespace mk;
+
+//TEST_CASE("Synchronous whatsapp test") {
+//    test::nettests::make_test<WhatsappTest>("meek_fronted_requests.txt")
+//        .run();
+//}
```

Comment:
You should uncomment this. You don't need to provide any filepath argument in this case, AFAICT.

Reply:
acked